src/electionguard_cli/encrypt_ballots/encrypt_ballots_election_builder_step.py

```diff
@@ -10,8 +10,12 @@ class EncryptBallotsElectionBuilderStep(ElectionBuilderStep):
     def build_election_with_context(
```

Comment:
Why is this identical function in two files?

Reply:
They do identical things but take distinct input params (`ImportBallotInputs` vs `EncryptBallotInputs`). I tried renaming them, but it felt wordy.  If you feel strongly about it please reopen and propose names.